control/adafruithat/planktoscope/camera/mqtt.py

```diff
@@ -89,14 +89,11 @@ def run(self) -> None:
         streaming_thread.start()
 
         loguru.logger.info("Starting the MQTT backend...")
-        # TODO(ethanjli): expose the camera settings over "camera/settings" instead! This requires
-        # removing the "settings" action from the "imager/image" route which is a breaking change
-        # to the MQTT API, so we'll do this later.
-        mqtt = messaging.MQTT_Client(topic="imager/image", name="imager_camera_client")
-        # TODO(ethanjli): allow an MQTT client to trigger this broadcast with an MQTT command. This
-        # requires modifying the MQTT API (by adding a new route), and we'll want to make the
-        # Node-RED dashboard query that route at startup, so we'll do this later.
-        mqtt.client.publish("status/imager", json.dumps({"camera_name": self._camera.camera_name}))
+        mqtt = messaging.MQTT_Client(topic="camera/info", name="imager_camera_client")
+        mqtt.client.publish(
+            "status/camera/info",
+            json.dumps({"status": "success", "camera_name": self._camera.camera_name}),
+        )
```

Comment:
To keep things simple, we do not need to push the camera name to all MQTT clients here; instead, we can just wait for the Node-RED dashboard to request the camera name.